allennlp/models/model.py

```diff
@@ -109,21 +109,7 @@ def forward_on_instance(self, instance: Instance, cuda_device: int) -> Dict[str,
         and returns the result.  Before returning the result, we convert any ``torch.autograd.Variables``
         or ``torch.Tensors`` into numpy arrays and remove the batch dimension.
         """
-        instance.index_fields(self.vocab)
```

Comment:
👍

Reply:
This actually drives home @joelgrus's earlier point that maybe we should just have the caller do this, instead of having another method for it.  But that's ok, we don't want to break the API at this point.